File: Java/Processor.java
class Processor{
    private String name;
    private int price;

    public Processor(String name, int price){
        this.name = name;
        this.price = price;
    }

    public Processor(){
    }

    public String getName(){
        return name;
    }
    
    public void setName(String name){
        this.name = name;
    }

    public int getPrice(){
        return price;
    }

    public void setPrice(int price){
        this.price = price;
    }

    public void outs(){
        System.out.println("\nNama Processor: " + this.getName());
        System.out.println("Harga Processor: " + this.getPrice());
    }
}

File: Python/Processor.py
class Processor():
    #private members
    __name = "Lorem 3000"
    __price = 1000

    #Construct
    def __init__(self, name = "lorem", price = 1000):
        self.__name = name
        self.__price = price

    #Get-set
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

    #Print Output
    def outs(self):
        print("Nama Processor: %s" % self.__name)
        print("Harga Processor: %d" % self.__price)

    
